hack/update-codegen.sh

```diff
@@ -72,7 +74,7 @@ function generate_group() {
   echo "generating conversions"
   bazel run @io_k8s_code_generator//cmd/conversion-gen -- --input-dirs ${APIS_PKG}/${GROUP_NAME} --input-dirs ${APIS_PKG}/${GROUP_NAME}/${VERSION} --output-base ${GEN_TMPDIR}
 
-  rsync -a "${GEN_TMPDIR}/k8s.io/cluster-registry/" "${SCRIPT_BASE}/${SCRIPT_PACKAGE}"
+  rsync -a "${GEN_TMPDIR}/k8s.io/cluster-registry/" "${SCRIPT_BASE}/${REPO_DIRNAME}"
 
   if [[ "${GEN_TMPDIR}" == "/tmp/gen"* ]]; then
     rm -rf "${GEN_TMPDIR}"
```

Comment:
We should also add this to the `clean_up` exit signal handler in the event that any command fails.

Reply:
Yes, that makes sense. I've fixed this in the follow-up I'm working on.